android/libraries/rib-base/src/main/java/com/badoo/ribs/android/recyclerview/RecyclerViewHostRouter.kt

```diff
@@ -0,0 +1,43 @@
+package com.badoo.ribs.android.recyclerview
+
+import android.os.Bundle
+import android.os.Parcelable
+import com.badoo.ribs.android.recyclerview.RecyclerViewHostRouter.Configuration
+import com.badoo.ribs.android.recyclerview.RecyclerViewHostRouter.Configuration.Content
+import com.badoo.ribs.android.recyclerview.RecyclerViewHostRouter.Configuration.Content.Default
+import com.badoo.ribs.android.recyclerview.client.RecyclerViewRibResolver
+import com.badoo.ribs.core.Router
+import com.badoo.ribs.core.routing.action.RoutingAction
+import com.badoo.ribs.core.routing.action.RoutingAction.Companion.noop
+import com.badoo.ribs.core.view.RibView
+import kotlinx.android.parcel.Parcelize
+import java.util.UUID
+
+internal class RecyclerViewHostRouter<T : Parcelable>(
+    savedInstanceState: Bundle? = null,
+    private val feature: RecyclerViewHostFeature<T>,
+    private val ribResolver: RecyclerViewRibResolver<T>
+): Router<Configuration, Nothing, Content, Nothing, RibView>(
+    savedInstanceState = savedInstanceState,
+    initialConfiguration = Default,
+    permanentParts = emptyList()
+) {
+    sealed class Configuration : Parcelable {
+        sealed class Content : Configuration() {
+            @Parcelize object Default : Content()
+            @Parcelize data class Item(val uuid: UUID): Content()
```

Comment:
maybe let's use `String` id (or custom id class) instead of `UUID` class?

Reply:
why?

Reply:
I wouldn't tie our implementation to UUID.
Custom `Id` class will be much more manageable if we will have to change it in the future.

Reply:
will do later together with other places using this, now it's not critical